isolate/package.json

```diff
@@ -31,25 +31,29 @@
   "typings": "lib/cjs/index.d.ts",
   "types": "lib/cjs/index.d.ts",
   "dependencies": {
-    "xstream": "^11.7.0",
-    "@cycle/run": "^4.4.0"
+    "@cycle/run": "^4.4.0",
+    "symbol-observable": "^1.2.0",
```

Comment:
dependency, not devDependency? (I know this is probably coming from rxjs@6 but it's unfortunate it became a dependency of `isolate`. No other way to solve this?

Reply:
it should be a devDependency, it's only for the tests, must have fat-fingered that one